src/sentry/migrations/0477_backfill_organization_mappings_via_outbox.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.28 on 2023-06-01 18:56
+
+from django.db import migrations
+
+from sentry.models import OutboxCategory, OutboxScope
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_org_mapping_via_outbox(apps, schema_editor):
+    Organization = apps.get_model("sentry", "Organization")
+    RegionOutbox = apps.get_model("sentry", "RegionOutbox")
+
+    for org in RangeQuerySetWrapperWithProgressBar(Organization.objects.all()):
+        RegionOutbox(
+            shard_scope=OutboxScope.ORGANIZATION_SCOPE,
+            shard_identifier=org.id,
+            category=OutboxCategory.ORGANIZATION_UPDATE,
+            object_identifier=org.id,
+        ).save()
```

Comment:
This is a bit of a bummer, as I couldn't just use the static method available on the model for some reason.

Reply:
The reason is that within Django migrations you don't have access to the 'real' model class. Instead you get a generated 'historical' model which only contains methods from the ORM classes.